README.md

```diff
@@ -413,21 +411,26 @@ Once you are done with the development, you can deploy the service using any of
 As the first step, you can build a Ballerina executable archive (.balx) of the service that you developed above. Navigate to `/content-based-routing/guide` and run the following command. 
 
 ```bash
-   $ ballerina build company_recruitment_agency_service
+   $ ballerina build
 ```
 
 Once the `company_recruitment_agency_service.balx` file is created inside the target folder, you can run that with the following command. 
 
 ```bash
+   $ ballerina run target/company_data_service.balx 
    $ ballerina run target/company_recruitment_agency_service.balx
 ```
 
-The successful execution of the service will show us the following output. 
+The successful execution of the service will show us the following outputs. 
 
 ```
-   ballerina: initiating service(s) in 'target/company_recruitment_agency_service.balx'
-	ballerina: started HTTP/WS endpoint 0.0.0.0:9091
-	ballerina: started HTTP/WS endpoint 0.0.0.0:9090
+    ballerina: initiating service(s) in 'target/company_recruitment_agency_service.balx'
```

Comment:
can you verify this message please? I believe it has to be something like "Initiating service(s) in 'target/company_recruitment_agency_service.balx'"